tests/python/pants_test/backend/jvm/tasks/jvm_compile/scala/test_scalac_plugin_integration.py

```diff
@@ -74,3 +75,22 @@ def test_plugin_uses_other_plugin(self):
       }
     }
     self._do_test(['args', 'from', 'target', 'local'], config, 'local')
+
+  def test_compiler_option_sets_required_scalac_plugins(self):
+    self._do_test_global(
+      args=[],
+      extra_config={
+        'compile.rsc': {
+          'compiler_option_sets_enabled_args': {
+            'option-set-requiring-scalac-plugin': [
+              '-S-P:simple_scalac_plugin:abc',
+              '-S-P:simple_scalac_plugin:def',
+            ],
+          },
+          'compiler_option_sets_required_scalac_plugins': {
```

Comment:
For symmetry, perhaps this should be `compiler_option_sets_enabled_scalac_plugins` ?

Reply:
That's great! Will do!